docs/static-pvc.md

```diff
@@ -117,3 +124,126 @@ persistentvolumeclaim/fs-static-pvc created
 
 **Note** deleting PV and PVC doesnot deleted the backend rbd image, user need to
 manually delete the rbd image if required
+
+## CephFS static PVC
+
+CephFS sub-volume created manually can be mounted and unmounted to an app,
+below steps show how to create a CephFS volume, static PV and static PVC.
+
+### Create CephFS sub-volume
+
+If you already have a CephFS sub-volume created and contains some data
+which you want to access by the application pod you can skip this step.
+
+Lets create a new CephFS sub-volume of size 1 GiB in ceph cluster which
+we are going to use for static PVC, before that we need to create
+the subvolumegroup. **myfs** is the file system name(volume name) inside
+which subvolume should be created.
+
+```bash
```

Comment:
if there is no output from command use console instead of bash. need same changes in other places also